src/server/index.js

```diff
@@ -36,6 +36,8 @@ console.log('Acting as faucet for address:', config.address)
 //
 
 // ProviderEngine based caching layer, with fallback to geth
+// 
+// need to be able to dynamically change rpcOrigin based on network requesting funds
```

Comment:
the provider stack is quite, old it should be replace by https://github.com/MetaMask/eth-json-rpc-middleware and the json-rpc-engine stack

Reply:
> the provider stack is quite, old it should be replace by https://github.com/MetaMask/eth-json-rpc-middleware and the json-rpc-engine stack

@kumavis Will that change the syntax here much?